cnm/plugin.go

```diff
@@ -85,12 +85,12 @@ func (plugin *Plugin) EnableDiscovery() error {
 
 	// Create the spec directory.
 	path := plugin.getSpecPath()
-	os.MkdirAll(path, 0755)
+	os.MkdirAll(path, 0o755)
```

Comment:
I am inclined towards checking errors on mkdirall. wdyt?

Reply:
i agree, but not in this PR